data/report_template.html

```diff
@@ -297,13 +278,13 @@
 
             // determine if we will be doing any internal counting
             var keys = [];
-            if ($("#show_activities").attr("checked"))
+            if (document.getElementById("show_activities").checked)
```

Comment:
I believe you should keep the jquery syntax but use the `.prop("checked")` function or `.checked` attribute.
```suggestion
            if ($("#show_activities").checked))
```
Same goes for the two other occurrences. Not tested but it should work AFAIK.

Reply:
Actually I did test my change now and it doesn't work... not sure why though. I was pretty convinced that I was correct after having read https://api.jquery.com/prop/

Apparently it's been a long time that I haven't used jQuery... I notice though that we have a very old jQuery (1.4.1 vs 3.5.1), maybe it's partly related.

Reply:
```suggestion
            if ($("#show_activities")[0].checked)
```

This one is tested and works!